src/Menu/MenuBar.hpp

```diff
@@ -28,6 +28,10 @@ Copyright_License {
 
 #include <tchar.h>
 
+
+const unsigned int vertical_button_scale = 6;
```

Comment:
Should be `static constexpr` and needs documentation. And a better name - this is a global variable, after all.